This patch will enable Go's position-independent executeable buildmode when
building binaries via Makefile.


Signed-off-by: Camber Huang<camber@aosc.io>
---
diff --git a/Makefile b/Makefile
index 5a17dae..b29f976 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 all: popub-local/popub-local popub-relay/popub-relay
 
 PREFIX=/usr/local
-GOBUILD=go build
+GOBUILD=go build --buildmode=pie
 
 clean:
        rm -f popub-local/popub-local popub-relay/popub-relay
